primitives/src/constants.rs

```diff
@@ -0,0 +1,39 @@
+// This file is part of Hydradx-node.
+
+// Copyright (C) 2020-2021  Intergalactic, Limited (GIB).
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod chain {
+	pub use crate::{AssetId, Balance};
+	pub use frame_support::weights::{constants::WEIGHT_PER_SECOND, Weight};
+
+	/// Core asset id
+	pub const CORE_ASSET_ID: AssetId = 0;
+
+	/// Max fraction of pool to buy in single transaction
+	pub const MAX_OUT_RATIO: u128 = 3;
+
+	/// Max fraction of pool to sell in single transaction
+	pub const MAX_IN_RATIO: u128 = 3;
+
+	/// Trading limit
+	pub const MIN_TRADING_LIMIT: Balance = 1000;
+
+	/// Minimum pool liquidity
+	pub const MIN_POOL_LIQUIDITY: Balance = 1000;
+
+	/// We allow for
+	pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND / 2;
```

Comment:
0.5 sec is limitation of parachain, solochain is fine with 2 sec processing time

```suggestion
	pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND * 2;
```